docs/robots/ridgeback/tutorials_ridgeback.mdx

```diff
@@ -1,12 +1,1080 @@
+
+  This test:
+
+  - Subscribes to the `/imu/data` ROS topic to receive angular velocity measurements from the IMU's Gyroscope.
+    These measurements are converted into angular displacement estimations, and the robot will rotate until 2 full revolutions are estimated.
+  - Subscribes to the `/odometry/filtered` ROS topic to receive angular velocity estimations from the EKF odometry.
+    These measurements are converted into angular displacement estimations, and are output as comparison to the angular displacement estimations from the IMU's Gyroscope.
+  - Publishes to the `/cmd_vel` ROS topic to send drive commands to rotate the robot.
+  - The user will be asked to verify that the robot rotates 2 full revolutions.
+
+  :::note
+
+  The **Rotate Test** rotates the robot using the IMU's Gyroscope data, which inherently will not be 100% accurate.
+  Therefore, some undershoot/overshoot is to be expected.
+
+  :::
+
+- **Drive Test**
+
+  The **Drive Test** drives the robot forward 1 metre and checks that the motors, encoders, and encoder-fused odometry are working properly.
+
+  This test:
+
+  - Subscribes to the `/ridgeback_velocity_controller/odom` ROS topic to receive linear displacement estimations from the encoder-fused odometry.
+    The robot will drive forward until 1 metre is estimated.
+  - Subscribes to the `/feedback` ROS topic to receive linear displacement measurements from the individual encoders.
+    These measurements are output as comparison to the linear displacement estimations from the encoder-fused odometry.
+  - Subscribes to the `/joint_state` ROS topic to receive linear displacement measurements from individual the encoders.
+    These measurements are output as comparison to the linear displacement estimations from the encoder-fused odometry.
+  - Publishes to the `/cmd_vel` ROS topic to send drive commands to drive the robot.
+  - The user will be asked to verify that the robot drives forward 1 metre.
+
+  :::note
+
+  The **Drive Test** drives the robot using the Odometry data, which inherently will not be 100% accurate.
+  Therefore, some undershoot/overshoot is to be expected.
+
+  :::
+
+- **Cooling Test**
+
+  The **Cooling Test** checks that the fans are working properly.
+
+  This test makes the fans spin at different speeds by publishing fan speed commands to the `/mcu/cmd_fans` ROS topic.
+  The user will be asked to verify that the fans change to the expected speeds.
+
+### CAN Bus Test
+
+The `check_can_bus_interface` script checks that communication between the motors, encoders, robot's MCU, and robot's computer are working properly over the CAN bus interface.
+
+This script verifies that the `can0` interface is detected and activated, then proceeds to check the output of `candump` to verify that good CAN packets are being transmitted.
+Based on the Ridgeback configuration, either Ridgeback-D or Ridgeback-O, this script will know to check for good CAN packets from 2 or 4 encoders, respectively.
```

Comment:
Is this carried over from Dingo? I don't think there is a Ridgeback-D and Ridgeback-O.

Reply:
Good catch. Removed.